ref/config/serverConfiguration.adoc

```diff
@@ -221,37 +175,28 @@ operator can be one of `+`, `-`, `*`, `/` for example:
 <variable name="threeagain" value="${six/two}" />
 ----
 
-There are a number of predefined variables:
+The following predefined variables are supported:
```

Comment:
I don't think "are supported" is correct. They exist, they are built in. Are supported optionality that doesn't exist.

Reply:
Updated